### PR TITLE
actualizeFeed return number of new articles as 3rd

### PR DESCRIPTION
**Pull request by sjeon87**
I've modified `actualizeFeed()` to return the number of new articles as the 3rd item of an array, `cli/actualize-user.php` to take and print it out.

It might be helpful if someone wants to see how many new articles are fetched via  command line interface. (e..g, Making a cron job to actualize feeds and send a notification mail regularly.)

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks for the pull request. Could you please fix the patch so it can target our /dev branch?

**Comment by sjeon87** [comment]
Thanks for response. I've resolved conflict in `feedController.php`

**Comment by Alkarex** [comment]
Please add yourself to the credits file :-)
https://github.com/FreshRSS/FreshRSS/blob/dev/CREDITS.md

**Comment by sjeon87** [comment]
Many thanks for your considerations.

File: CREDITS.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ People are sorted by name so please keep this order.
 * [dswd](https://github.com/dswd): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:dswd)
 * [ealdraed](https://github.com/ealdraed): [contributions](https://github.com/FreshRSS/FreshRSS/commits?author=ealdraed)
 * [Frans de Jonge](https://github.com/Frenzie): [contributions](https://github.com/FreshRSS/FreshRSS/commits?author=Frenzie), [Web](http://fransdejonge.com/)
+* [gsongsong](https://github.com/gsongsong): [contribuitions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:gsongsong)
```
Comment (Alkarex) [inline CREDITS.md]: Please fix typo at *contributions*

**Comment by sjeon87** [comment]
Oops, sorry. Since I am editing it via GitHub web interface, hard to use `commit --amend` or `squash` of `rebase`. Could you hold it for about a day and may I rebase the last two commits and push forcibly?

**Comment by Alkarex** [comment]
Don't worry too much about the git history, as long as the pull request is good.

**Comment by Alkarex** [comment]
Could you please add your third variable to these call as well?
* https://github.com/gsongsong/FreshRSS/blob/fd5bd80af18925b910c563657418ccba45eb388f/app/Controllers/feedController.php#L485
* https://github.com/gsongsong/FreshRSS/blob/fd5bd80af18925b910c563657418ccba45eb388f/p/api/pshb.php#L139

**Comment by sjeon87** [comment]
I'll do it tomorrow (here, Korea is late night).

But just for curious, AFAIK, PHP does not complain about fewer number of variables than expected. (At least my case of PHP 7.x) Is it for version-specific safety?
And it seems that `actualizeAction` and `pshb.php` are not going to be use `$nb_new_articles`. Is it just for consistency?

I just wanted to know what technical issues may exist or what philosophy you have behind it.

**Comment by Alkarex** [comment]
Ok :-) Yes, it is just for consistency, and to avoid taking risks with future versions of PHP, which might get more strict (as we have seen already with PHP 7.0 and 7.1 on several points).

**Pull request merged**